.github/CODEOWNERS

```diff
@@ -13,3 +13,5 @@
 #
 
 * @open-telemetry/dotnet-instrumentation-approvers
+*.md @theletterf @open-telemetry/dotnet-instrumentation-approvers
+docs/ @theletterf @open-telemetry/dotnet-instrumentation-approvers
```

Comment:
does it have to be here as well?

> This CODEOWNERS file contains errors

Reply:
in my previous attempts of CODEOWNERS if this reports error it doesn't work as intended. I think that we have to add @theletterf to some group or give some rights to this can work properly.

Reply:
> The people you choose as code owners must have write permissions for the repository. 

from: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

Maybe it would be simper just go create a `open-telemetry/dotnet-instrumentation-tech-writers` or `open-telemetry/dotnet-instrumentation-doc-writers` team?

@Kielek @pjanotti  WDYT?

Reply:
I am fine with both solutions.

Tested on my PR, which is expected outcome:
```
Ignoring error: RequestError [HttpError]: Reviews may only be requested from collaborators. One or more of the users or teams you specified is not a collaborator of the Kielek/opentelemetry-dotnet-instrumentation repository.
    at /home/runner/work/_actions/dyladan/component-owners/main/webpack:/component-owners/node_modules/@octokit/request/dist-node/index.js:86:1
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  status: 422,
  response: {
    url: 'https://api.github.com/repos/Kielek/opentelemetry-dotnet-instrumentation/pulls/344/requested_reviewers',
    status: 422,
    headers: {
      'access-control-allow-origin': '*',
      'access-control-expose-headers': 'ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset',
      connection: 'close',
      'content-length': '300',
      'content-security-policy': "default-src 'none'",
      'content-type': 'application/json; charset=utf-8',
      date: 'Tue, 27 Sep [20](https://github.com/Kielek/opentelemetry-dotnet-instrumentation/actions/runs/3133257794/jobs/5086441437#step:2:21)22 05:52:58 GMT',
      'referrer-policy': 'origin-when-cross-origin, strict-origin-when-cross-origin',
      server: 'GitHub.com',
      'strict-transport-security': 'max-age=31536000; includeSubdomains; preload',
      vary: 'Accept-Encoding, Accept, X-Requested-With',
      'x-content-type-options': 'nosniff',
      'x-frame-options': 'deny',
      'x-github-media-type': 'github.v3; format=json',
      'x-github-request-id': '0406:5DBC:1A6C075:1B2F9CE:63328FBA',
      'x-ratelimit-limit': '1000',
      'x-ratelimit-remaining': '987',
      'x-ratelimit-reset': '1664261[21](https://github.com/Kielek/opentelemetry-dotnet-instrumentation/actions/runs/3133257794/jobs/5086441437#step:2:22)3',
      'x-ratelimit-resource': 'core',
      'x-ratelimit-used': '13',
      'x-xss-protection': '0'
    },
    data: {
      message: 'Reviews may only be requested from collaborators. One or more of the users or teams you specified is not a collaborator of the Kielek/opentelemetry-dotnet-instrumentation repository.',
      documentation_url: 'https://docs.github.com/rest/reference/pulls#request-reviewers-for-a-pull-request'
    }
  },
  request: {
    method: 'POST',
    url: 'https://api.github.com/repos/Kielek/opentelemetry-dotnet-instrumentation/pulls/344/requested_reviewers',
    headers: {
      accept: 'application/vnd.github.v3+json',
      'user-agent': 'octokit-core.js/3.5.1 Node.js/12.[22](https://github.com/Kielek/opentelemetry-dotnet-instrumentation/actions/runs/3133257794/jobs/5086441437#step:2:23).7 (linux; x64)',
      authorization: 'token [REDACTED]',
      'content-type': 'application/json; charset=utf-8'
    },
    body: '{"reviewers":["theletterf"]}',
    request: { agent: [Agent], hook: [Function: bound bound register] }
  }
}
```

I would like to give it a chance, if it will not work I will revert it.

Reply:
Let's give a try as it is.

Reply:
> Maybe it would be simper just go create a open-telemetry/dotnet-instrumentation-tech-writers or open-telemetry/dotnet-instrumentation-doc-writers team?

If it doesn't work we pursue this route. IIRC it is outside our own rights to create such team.